doc/developers/contributing.rst

```diff
@@ -779,6 +788,56 @@ cross-compatibility and is included in scikit-learn as
 ``sklearn.externals.six``.
 
 
+.. _code_review:
+
+Code Review Guidelines
+======================
+Reviewing code contributed to the project as PRs is a crucial component
+of scikit-learn development. We encourage anyone to start reviewing code of other
+developers. The code review process is often highly educational for everybody involved.
```

Comment:
"This is particularly appropriate if it is a feature you would like to use, and so can respond critically about whether the PR meets your needs."